### DISCUSSION TITLE
New "Themes" settings

### DISCUSSION THREAD

**Discussion by sadellie** [sadellie/unitto#37] [Show and tell]
Yes, you will be able to choose color scheme. Icon settings will be added a bit later.

<img src="https://user-images.githubusercontent.com/10384356/230667279-e5ebabb3-0853-4338-b2a7-7984e53b3702.png" width="45%" />

**Comment by sadellie** [comment]
App icon setting. Not final result, will add more colors + UI tweaks.
<img src="https://user-images.githubusercontent.com/10384356/230793240-033d764c-5ea8-4435-8bfb-efa678b4f423.png" width="45%" />

**Reply by ghost** [reply]
Looks perfect 🥳

**Reply by e-t-l** [reply]
Did this ever get implemented? I don't see any options for icon theme in the app. (Main thing I'd really is just an option to use a permanent darkmode icon, so it doesn't have the bright white background. 

Could be as simple as bundling a black-background icon in the APK and letting the user handle it when customizing their homescreen/launcher app. 

Another option would be to have the app include a "shortcut" with the same `/.MainActivity` target, but the icon for it is alt/darkmode.
